src/vs/workbench/contrib/relauncher/browser/relauncher.contribution.ts

```diff
@@ -26,7 +26,7 @@ interface IConfiguration extends IWindowsConfiguration {
 	debug?: { console?: { wordWrap?: boolean } };
 	editor?: { accessibilitySupport?: 'on' | 'off' | 'auto' };
 	security?: { workspace?: { trust?: { enabled?: boolean } } };
-	window: IWindowSettings & { experimental?: { windowControlsOverlay?: { enabled?: boolean }; useSandbox?: boolean } };
+	window: IWindowSettings & { experimental?: { windowControlsOverlay?: { enabled?: boolean } } };
```

Comment:
I am removing sandbox from the relauncher because I fear that an enabled experiment would tell the user to restart which is odd.

Reply:
I would prefer to have this in relaunched so that users know that they have to restart when they change it. You can always check for the source of the change so that you do not prompt restart if the change is coming from default source.

Reply:
Brought back.